packages/jest-util/src/isInteractive.ts

```diff
@@ -7,4 +7,8 @@
 
 import isCI = require('is-ci');
 
-export default !!process.stdout.isTTY && process.env.TERM !== 'dumb' && !isCI;
+export default process &&
```

Comment:
? Seems like something that should be in a separate PR

Reply:
We use `deepCyclicCopy` in `jest-matcher-utils` by importing `jest-util`.
Therefore, in browser testing, it will [fail](https://circleci.com/gh/facebook/jest/77286) without checking `process` or `stream`.

Another solution is just import `deepCyclicCopy` only.